doc/cloud/cloud-deploy.rst

```diff
@@ -0,0 +1,268 @@
+Deploy
+======
+
+This section will help you to deploy the KNoT Cloud.
+
+----------------------------------------------------------------
+
+Installation and usage
+----------------------
+
+Stacks for development and production environments are provided in order to assist the user's needs. The development stack must be used if one needs to modify any of the components of the stack. A command line tool will download the source for each component and plug it into the containers, which all have *hot reload* enabled. The production stack must be used in all other cases.
```

Comment:
Please, use a maximum of 79 characters per line.

Reply:
Done.